### PR TITLE
23 author affiliation

### LINKED ISSUE #23: Author affiliation
It may be useful to list author affiliation in the article list.

### PR DESCRIPTION
**Pull request by Suhyma**
Adding author affiliation for the corresponding author only, and making sure no author names are duplicated in the list by doing so. In the API response we get, the author list is in the form of `lastName, initials` but the corresponding author is listed by their full name. So, a bit of filtering and comparison is involved to make sure we have the right author's affiliation linked. Some cases I'm considering here that would affect the process of searching for the corresponding author name in the full author list:

- Authors who have the same last name
- Authors with accents in their name
- Authors who have last names of multiple words separated by spaces (not hyphenated, like "Pryce Roberts, A.")

### CODE REVIEW

File: src/dashboard/category-results-screen.js [hunk 1]
```diff
@@ -4,16 +4,76 @@ import CategoryCard from './category-card.js';
...
-  return h('div', { class: 'paper' }, [
+  const corrAuthor = findCorrespondingAuthor(paper).toString();
+
+  const paperElements = [
     h('a', { href: paper.finalURL, target: '_blank', class: 'paper-link' }, [
       h('div', { class: 'paper-read' }),
       h('div', { class: 'paper-title' }, paper.title)
     ]),
-    h('div', { class: 'paper-authors' }, paper.authors),
+    h('div', { class: 'paper-authors' }, finalizeAuthorsList(paper.authors, corrAuthor), [
```
Comment (Suhyma) [inline src/dashboard/category-results-screen.js:14]: Does the order of the authors matter if we're planning to condense the author list? Currently I've set it up so the corresponding author with their institution tooltip is just tacked onto the end of the author list.

**Comment by jvwong** [comment]
There's errors when I try to load a results screeen:

Screen Shot 2023-04-21 at 10 23 59 AM

File: src/dashboard/category-results-screen.js (hunk 1 above)
Comment (jvwong) [reply src/dashboard/category-results-screen.js:14]: The order of the authors is important. Typically first (few) author(s) performed the day to day research; last (few) author(s) usually reflect the supervisor, lab head etc.

File: src/dashboard/category-results-screen.css [hunk 2]
```diff
@@ -81,4 +81,41 @@
   top: 50%;
   margin-top: -0.25em;
   margin-left: -1.125em;
-}
\ No newline at end of file
+}
+
+.tooltip {
```
Comment (jvwong) [inline src/dashboard/category-results-screen.css:86]: I wouldn't recommend burying info in tooltips. Also this gets more complicated in emails, mobile etc.

**Comment by maxkfranz** [comment]
Is there a screenshot preview of the change available somewhere?

**Comment by maxkfranz** [comment]
Also: The preview page isn't working for me: https://23-author-affiliation.hyper-recent.pages.dev/

**Comment by Suhyma** [comment]
Sorry about that, I forgot to commit a small change I made yesterday. The error should be fixed now!

**Review by jvwong — APPROVED** [review]

**Pull request merged**